### ISSUE TITLE
Incompatability with Bob's Mods v0.13

### ISSUE THREAD

**Issue by DRY411S** [DRY411S/Recycling-Machines#8]
Some of Bob's Mods recipes are not being reversed properly and causing the game to crash because 'bad' recipes have been produced. Example error message:

```
Error while loading recipe prototype "dry411srev-electric-mining-drill" 
(recipe): No such node (name)
Modifications: ZRecycling

```

**Comment by DRY411S** [comment]
Fixed in v0.13.11

**Comment by DRY411S** [comment]
Fixed in v0.12.36
